### DISCUSSION TITLE
unable to fretch file error during upgrading

### DISCUSSION THREAD

**Discussion by balookhan67** [termux/termux-packages#12844] [Q&A] [answered]
Please may anyone help me 
Whenever i upgrade files it slow downs my network and show me unable to fetch files especially in fetching python files 
fe![IMG_20221104_035343.jpg](https://user-images.githubusercontent.com/64676458/199851024-53eacf36-5790-430d-8bae-983ea3a9059c.jpg)

**Comment by sylirre** [comment, marked as answer]
Use `termux-change-repo` to select another mirror.

Tips:
* **space** button is to *select item*
* **enter** button is to *confirm current selection*
